src/m3u8/m3u8-parser.js

```diff
@@ -436,7 +440,8 @@
               // setup an encryption key for upcoming segments
               key = {
                 method: entry.attributes.METHOD || 'AES-128',
-                uri: entry.attributes.URI
+                uri: entry.attributes.URI,
+                iv: entry.attributes.IV || null
```

Comment:
For most of the other properties in this parser, we haven't added key/values for attributes that weren't present in the m3u8.